bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -541,4 +542,12 @@ private void normalizePackagePrefixes(WindupConfiguration windupConfiguration)
 
         return packages;
     }
+
+    private static boolean isJavaArchive(Path path)
```

Comment:
This should use org.jboss.windup.util.ZipUtil#getZipExtensions rather than its own implementation.